css-tutorial.md

```diff
@@ -0,0 +1,483 @@
+```
+The above command will append the triple `<ex:s3> <ex:p3> <ex:o3>.` to the RDF resource,
+which can be verified by GETting http://localhost:3000/a/b/c .
+
+#### DELETE a resource
+Finally, DELETE can be used to remove resources.
+
+```shell
+curl -X DELETE http://localhost:3000/a/b/c
+```
+
+This will completely remove the resource from the server.
+The parent containers will remain,
+so http://localhost:3000/a/ and http://localhost:3000/a/b/ still exist.
+
+### Using a different configuration
+The server we have been using so far has been running with a memory backend,
+which means all data will be lost when the server is stopped.
+There are many ways to configure CSS and several default configurations are provided that can be used.
+When starting the server with `npm start` the server will default to a memory-based configuration.
+
+#### File backend
+To have a more permanent storage solution, we will now start the server with a file backend.
+First stop the server that is still running, 
+then start it again with the following command from the source folder:
+```shell
+npm start -- -c config/file.json -f ../.data
+```
+The above command will start the server with the `file.json` config,
+one of the available [default configurations](https://github.com/solid/community-server/tree/main/config),
+which starts the server with a file backend.
+The `-f ../.data` parameter tells the server we want all data to be stored in that folder. 
+
+## Web Access Control (WAC)
+In the previous HTTP examples no authentication was required to modify any of the resources,
+but in practice you probably want to control who is allowed to read and modify data.
+CSS (and most other Solid servers) make use of 
+[Web Access Control (WAC)](https://solid.github.io/web-access-control-spec/)
+to restrict access to resources.
+
+First we will set up the server after which we will cover the basics of this authentication scheme.
+
+### Setup
+With the newly configured server,
+if you browse to http://localhost:3000/ you will now see a setup screen.
+This setup allows you already set some initial permissions on the server.
+
+For this tutorial we will choose the `Sign up for an account` option.
+Choose the options to create a new WebID in the root,
+more on what this means later.
+Make sure you don't forget the email/password combination that you choose.
+The response page will show some extra information about the setup result.
+
+#### WebID
+In the previous step the server created several resources for you,
+including a profile document with a WebID.
+A [WebID](https://dvcs.w3.org/hg/WebID/raw-file/tip/spec/identity-respec.html)
+is a way to identify yourself on the Web.
+We will make use of this a bit later in this tutorial.
+After setup, you can find your profile resource at http://localhost:3000/profile/card ,
+with your WebID being http://localhost:3000/profile/card#me .
```

Comment:
Sometimes you put a space between an url and dot, and sometimes not.

Reply:
I wasn't sure if github was going to handle it correctly if there was no space. Will see what it did in the cases where I forgot a space.